BackendBench/scripts/main.py

```diff
@@ -243,7 +257,11 @@ def cli(
             results = evaluator.get_results()
 
         for result in results:
-            correctness_score = result.correctness_score
+            correctness_score = all(
```

Comment:
Can you also add a per@p on the operator level as well in test_data (I'd just do this in eval.py)

Reply:
ohh kernelbench just has a slew of tests and does fast_p on those. The way you are currently calculating perf@p is that you run it off the aggregated correctness scores / perf scores of ops  rather than all of the individual tests. So really we're calculating things off of the mean of the correctness / perf rather than on the test by test level.

Imo for test_dict we should also calculate it on the test by test level (this was can just call fast_p tbh as its equivalent)